playbooks/templates/k8s/namespace/01_matomo-config-map.yml.j2

```diff
@@ -190,10 +190,20 @@ data:
     waiUrl = "https://{{ wai_base_domain[item] }}"
 
     [QueuedTracking]
+    backend = redis
+    useSentinelBackend = 0
+    redisHost = ingestion-redis
+    redisPort = 6379
+    redisDatabase = 0
```

Comment:
Se questo funziona allora:
- Ingestion deve diventare uno statefulset in modo da avere l'id del container (0..7)
- Potremmo utilizzare redisDatabase differenti, 0 e 1, facendo modificare il valore dall'initContainer come calcolo, ad esempio da *containerIndex* < (*numQueueWorker* / 2)  ? 0 : 1 e reimpostare *numQueueWorkers* a *numQueueWorkers* / 2 sempre tramite initContainer

In questo modo, per produzione, avremo due blocchi di ingestion e worker, ognuno composto da 4 elementi, che puntano a database differenti o, se volessimo, ad istanze redis differenti.

Reply:
Si ho provato e funziona, quindi potremmo iniziare a vedere come si comporta in ambiente play.
L'idea sarebbe quella di usare due istanze redis per dividere il carico. Usare un db differente della stessa istanza secondo me non cambia di molto le cose perché in fondo è solo un indice diverso ma sotto ci sono sempre le stesse risorse.

Reply:
Concordo sul fatto che il redisDatabase sia ininfluente e che quindi sia più corretto usare istanze Redis differenti. Vediamo come si comporta play su Region-3.